sklearn/covariance/empirical_covariance_.py

```diff
@@ -79,6 +80,8 @@ def empirical_covariance(X, assume_centered=False):
     else:
         covariance = np.cov(X.T, bias=1)
 
+    if covariance.ndim == 0:
```

Comment:
`np.dot` returns a scalar with `(m, 1)` and `(1, m)` array. As per the functions documentation a `(1, 1)` array should be returned.